lstchain/scripts/lstchain_find_pedestals.py

```diff
@@ -0,0 +1,173 @@
+#!/usr/bin/env python
+
+"""
+This script has to be launched (with no arguments) on a directory containing
```

Comment:
You mean "in a directory"? As in the CWD is taken? That's not good for e.g. running it via osa or in parallel on lots of directories. Please add an input directory argument.